postgresql-11.5/src/bin/psql/commandbteq.c

```diff
@@ -237,16 +239,16 @@ extract_token(char *command, char *delimiter, char **left, char **right) {
     }
     char *command_dup = strdup(command);
 
-    char *tmp;
-    tmp = strtok(command_dup, delimiter);
+    char *value;
```

Comment:
The previous comment about renaming `tmp` to `value` was not related to this part. It should be kept as it was.
Please, revert the last change.